### ISSUE TITLE
Help wanted: Getting "[Videomass]: FAILED ! " message without explanation/log

### ISSUE THREAD

**Issue by hakrek** [jeanslack/Videomass#136]
@jeanslack Hi there and thank you so much for creating Videomass! 
I'm on macOS 10.14.6, have ffmpeg 4.2.2 installed (I think?) and I'm trying to batch remove audio from video files (a single track of audio per video file). I keep getting this message at the end of the process: 

![Image 2022-06-22 at 9 16](https://user-images.githubusercontent.com/44825334/174917896-9a7dd9fe-b579-4fc7-8c56-199c71f19062.jpg)

Is it an ffmpeg or Videomass issue? Can you please help me understand how to deal with this? 

thanks

**Comment by jeanslack** [comment]
@hakrek 
to batch remove audio from video files what function did you use with Videomass? **Presets Manager** or **A/V Conversion** ?

Also, try to consult the Videomass logs based on the function used (menu bar -> View -> Show Logs)

Let me know

**Comment by hakrek** [comment]
@jeanslack thanks for your comment, I think you already helped me resolve this :) !
It seems I had used **Presets Manager** before and it didn't work. After you pointed it out, I tried doing it with **A/V Conversion**, by simply selecting :
Video/Video Encoder/Codec=Copy and 
Audio/Audio Encoder=No Audio

and boom, files were successfully processed in seconds. :) 
I'm posting the log below for reference. 

Thanks for your help, I really appreciate it!

<img width="977" alt="Screen Shot 2022-06-23 at 12 58 08" src="https://user-images.githubusercontent.com/44825334/175206558-5aa3800f-d187-4ca5-a15e-265e694ae8c0.png">
